playgrounds/astro/src/env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="astro/client" />
```

Comment:
I'm new to astro. What is the need of this file/line? Should this be `.gitignore`d, similar to [L6](https://github.com/iTwin/iTwinUI-react/pull/951/files#diff-33c4e506f435461adae7f2fd3139ab1c45d9ce24daff6307f4e132b32a785f74R6) in the next playground?

Reply:
it should not be gitignored. this adds types for astro and vite. for example when you type `Astro.` in a `.astro` file, you will get auto complete